contracts/credit/CreditFacadeV3.sol

```diff
@@ -288,32 +288,25 @@ contract CreditFacadeV3 is ICreditFacadeV3, ACLNonReentrantTrait {
     /// @param calls List of calls to perform before liquidating the account
     /// @dev When the credit facade is paused, reverts if caller is not an approved emergency liquidator
     /// @dev Reverts if `creditAccount` is not opened in connected credit manager
-    /// @dev Reverts if account is not liquidatable
+    /// @dev Reverts if account has no debt or is neither unhealthy nor expired
     /// @dev Reverts if remaining token balances increase during the multicall
     function liquidateCreditAccount(address creditAccount, address to, MultiCall[] calldata calls)
         external
         override
         whenNotPausedOrEmergency // U:[FA-2,12]
         nonReentrant // U:[FA-4]
     {
-        address borrower = _getBorrowerOrRevert(creditAccount); // U:[FA-5]
-
         uint256 skipCalls = _applyOnDemandPriceUpdates(calls);
 
         CollateralDebtData memory collateralDebtData =
-            ICreditManagerV3(creditManager).calcDebtAndCollateral(creditAccount, CollateralCalcTask.DEBT_COLLATERAL); // U:[FA-15]
-
-        bool isExpired;
-        {
-            bool isLiquidatable = collateralDebtData.twvUSD < collateralDebtData.totalDebtUSD; // U:[FA-13]
-            if (!isLiquidatable && _isExpired() && collateralDebtData.debt != 0) {
-                isLiquidatable = true; // U:[FA-13]
-                isExpired = true; // U:[FA-14]
-            }
-            if (!isLiquidatable) revert CreditAccountNotLiquidatableException(); // U:[FA-13]
+            ICreditManagerV3(creditManager).calcDebtAndCollateral(creditAccount, CollateralCalcTask.DEBT_COLLATERAL); // U:[FA-16]
+
+        bool isUnhealthy = collateralDebtData.twvUSD < collateralDebtData.totalDebtUSD;
+        if (collateralDebtData.debt == 0 || !isUnhealthy && !_isExpired()) {
+            revert CreditAccountNotLiquidatableException(); // U:[FA-13]
         }
```

Comment:
Looks much cleaner this way. Besides, `collateralDebtData.debt == 0` is now checked in all cases, not only in expired one.